src/api/app/models/bs_request.rb

```diff
@@ -994,6 +1004,27 @@ def required_checks
     target_project_objects.pluck(:required_checks).flatten.uniq
   end
 
+  # Collection of grouped open reviews
+  def grouped_reviews
+    groups = { by_user: [], by_group: [], by_project: [], by_package: [], by_staging_project: [] }
+    reviews.where(state: 'new').find_each do |review|
+      if review[:by_user]
+        groups[:by_user] << review
+      elsif review[:by_group]
+        groups[:by_group] << review
+      elsif review[:by_project]
+        if review[:by_project].include?('Staging')
```

Comment:
This is not how we can identify staging projects right?

Reply:
With `staging_project?` from the `StagingProject` concern, we can tell if a project is a staging project. It is if it has a staging workflow:

https://github.com/openSUSE/open-build-service/blob/2b85da11e69942b841d0189cb89771e2a0e6213f/src/api/app/models/concerns/staging_project.rb#L158-L160

Reply:
It would also be interesting if this a staging project of project the request is against.

Reply:
So my suggestion, push this feature out.

Reply:
It's updated. I'm using this method to check staging projects.